scripts/darwin/metrics.sh

```diff
@@ -30,4 +47,99 @@ else
   target_branch=$(echo "$data" | grep "Target branch is:" | awk '{print $4}')
 fi
 
+# Pathenon Metrics Application Analysis
+#
+# ********* Overview ********
+#
+# The below command at the bottome of the page s rather involved. Here the parthenon
+# metrics app is being told to analyze the results of the parthenon performance
+# regression tests. This consists of several steps:
+#
+# 1. performance test data must exist in the local parthenon repository
+#
+# 2. A comparison between the local test results and previous results must be conducted
+#    so that a regression can be spotted.
+#
+#      * In the case of a pull request, this means comparing the local results with results
+#        of the target branch.
+#
+#      * In the case of a scheduled run, where there is no merge, the local test results
+#        which are assumed to be the most up to date are compared with the performance of
+#        previous commits on the same branch.
+#
+#    The performance metrics of the non local repository are stored in the parthenon wiki.
```

Comment:
Please capitalize "Parthenon" everywhere. I think "Wiki" should also be capitalized...